integration/docker/Dockerfile

```diff
@@ -61,6 +67,10 @@ COPY --chown=${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} --from=extractor /opt/ /opt/
 COPY --chown=${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} conf /opt/alluxio/conf/
 COPY --chown=${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} entrypoint.sh /
 
+RUN if [ ${ENABLE_DYANMIC_USER} = "true" ] ; then \
```

Comment:
The journal is only accessed by Alluxio servers. Is there a case for running server containers not as root or Alluxio? Or is the requirement only for Alluxio client containers which can be a larger set of users?

Reply:
The user case is that:

every user in data scientist team has own NFS directory, and this directory is only accessible by his account. root  or alluxio account doesn't work. so alluxio container has to be run as the same user account.

Reply:
And I also wonder what's the real risk to make alluxio binary can be run by other users. And is there any simple sample for me to understand better?